portal/config/eproms/Organization.json

```diff
@@ -1981,6 +1981,34 @@
       },
       "resourceType": "Organization"
     },
+    {
+      "extension": [
+        {
+          "timezone": "Europe/London",
+          "url": "http://hl7.org/fhir/StructureDefinition/user-timezone"
+        },
+        {
+          "research_protocols": [
+			{"name": "IRONMAN v3"}
+          ],
+          "url": "http://us.truenth.org/identity-codes/research-protocol"
+        }
+      ],
+      "id": 1464499,
```

Comment:
Hmm, I get your numbering (to make unique), but this has me wondering if we shouldn't simply delete the old organization in place of the new.  Is there any value in keeping it around?  I get your concerns with test patients, but this should be inconsequential as a new org is taking its place.

In other words, how about we simply update the old organization's name, partOf and other altered bits?

Reply:
@pbugni  sounds good, will make that change